### PR TITLE
Tests for ESBJAVA-4565 fix

### PR DESCRIPTION
**Pull request by charithag**

### CODE REVIEW

File: modules/integration/tests-integration/tests-patches/src/test/resources/artifacts/ESB/sequence/esbjava4565/testSequence.xml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+<sequence xmlns="http://ws.apache.org/ns/synapse" name="testSequence">
+    <log>
+        <property name="testSequenceLog" value="testSequence called..."/>
+    </log>
+    <property
+            expression="get-property('registry','gov:/services/test/config/ftp.xml')"
+            name="ftpUrl" scope="default" type="OM"/>
```
Comment (DilanUA) [inline modules/integration/tests-integration/tests-patches/src/test/resources/artifacts/ESB/sequence/esbjava4565/testSequence.xml]: has some indentation issues.

File: modules/integration/tests-integration/tests-patches/src/test/resources/testng.xml [hunk 2]
```diff
@@ -220,5 +220,11 @@
         <!--</classes>-->
     <!--</test>-->
 
+    <test name="ESBJAVA_4565" preserve-order="true" verbose="2">
+        <classes>
+            <class name="org.wso2.carbon.esb.registry.task.ESBJAVA4565TestCase"/>
+        </classes>
+    </test>
+
```
Comment (DilanUA) [inline modules/integration/tests-integration/tests-patches/src/test/resources/testng.xml]: unnecessary new line

File: modules/integration/tests-integration/tests-patches/src/test/resources/artifacts/ESB/sequence/esbjava4565/testSequence.xml (hunk 1 above)
Comment (charithag) [inline modules/integration/tests-integration/tests-patches/src/test/resources/artifacts/ESB/sequence/esbjava4565/testSequence.xml]: This is the auto formatted xml by the idea with standard wso2 formatting guidelines. Attributes in property tag placed in below line as it is exceed the maximum column length.

File: modules/integration/tests-integration/tests-patches/src/test/resources/testng.xml (hunk 2 above)
Comment (charithag) [inline modules/integration/tests-integration/tests-patches/src/test/resources/testng.xml]: New line prior to last xml tag is already followed by the original file as it is improve the readability.

**Pull request merged**